src/dbos-runtime/config.ts

```diff
@@ -151,6 +151,12 @@ function prettyPrintAjvErrors(validate: ValidateFunction<unknown>) {
  * Considers DBOSCLIStartOptions if provided, which takes precedence over config file
  * */
 export function parseConfigFile(cliOptions?: DBOSCLIStartOptions, useProxy: boolean = false): [DBOSConfig, DBOSRuntimeConfig] {
+  if (cliOptions?.configfile) {
+    console.warn("The --configfile option is deprecated. Please use --appDir instead.")
+  }
+  if (cliOptions?.appDir) {
+    process.chdir(cliOptions.appDir)
```

Comment:
changing the current directory doesn't seem like the right approach. I'm interested what other folks think about this.

Reply:
I did it this way because I want reproducible behavior--an application launched with `--appDir` should behave exactly the same way as if you ran `npx dbos start` from its root directory. I'm worried anything else could make it really easy for bugs to slip in.

Reply:
Makes sense. What's the system state after running the command? Does this only change cwd for the DBOS process or does it also change for the calling process as well?

Reply:
As long as the calling process doesn't get it's CWD changed, I'm cool with this as is

Reply:
No, changing the working directory of a child process can never affect the parent as far as I know